src/styles/main-card.scss

```diff
@@ -0,0 +1,22 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  width: $width-set-card;
+  padding: 51px $side-padding 25px;
```

Comment:
where did you get those numbers ?
![image](https://user-images.githubusercontent.com/99331793/198034134-2c7c0cf1-f09a-4d1c-a00d-3f7e9ab9ec1d.png)
![image](https://user-images.githubusercontent.com/99331793/198034211-0a06798c-32f7-4acf-b4ed-84d5068b5b49.png)
![image](https://user-images.githubusercontent.com/99331793/198034519-880176f5-5707-41c2-9ae5-52bff4bf9047.png)

lets fix the tests

Reply:
![image](https://user-images.githubusercontent.com/94929618/198044804-9a511d8a-f72a-45b1-bc38-5782f9ec9a2e.png)